redis/client.py

```diff
@@ -3262,6 +3262,15 @@ def zscore(self, name, value):
         "Return the score of element ``value`` in sorted set ``name``"
         return self.execute_command('ZSCORE', name, value)
 
+    def zunion(self, keys, aggregate=None, withscores=False):
```

Comment:
What about the weights? https://redis.io/commands/ZUNION

Reply:
@chayim You can see in the test.
`_zaggregate` supports weights (can be provided as a dictionary with the `keys` argument).

Reply:
I will add it to the docstring